spark/v3.3/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkTable.java

```diff
@@ -57,4 +61,48 @@ public void testTableEquality() throws NoSuchTableException {
     Assert.assertNotSame("References must be different", table1, table2);
     Assert.assertEquals("Tables must be equivalent", table1, table2);
   }
+
+  @Test
+  public void testTableName() {
+    CatalogManager catalogManager = spark.sessionState().catalogManager();
+    TableCatalog catalog = (TableCatalog) catalogManager.catalog(catalogName);
+
+    for (String fileFormat : Lists.newArrayList("parquet", "orc", "avro")) {
+      for (String tableFormatVersion : Lists.newArrayList("1", "2")) {
```

Comment:
Now that the file format and Iceberg format version are no longer shown, it is not necessary to tests these.

Reply:
Done.